ReactSkia/sdk/OSKConfig.h

```diff
@@ -0,0 +1,37 @@
+ /*
+ * Copyright (C) 1994-2022 OpenTV, Inc. and Nagravision S.A.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+#ifndef OSK_CONFIG_H
+#define OSK_CONFIG_H
+
+// Dark Theme configuration
+#define OSK_DARK_THEME_BG_COLOR          SkColorSetARGB(0xFF, 0x34, 0x34, 0x34) //Dark Gray
+#define OSK_DARK_THEME_FONT_COLOR             SK_ColorWHITE
+#define OSK_DARK_THEME_PH_COLOR               SK_ColorDKGRAY
```

Comment:
instead of PH, can we mention as placeholder itself or add a comment mentioning as placeholder.